### ISSUE TITLE
delta-presence

### ISSUE THREAD

**Issue by mattiamazzoli** [vanderschaarlab/synthcity#306]
Hi,
regarding the delta-presence privacy metric, the Docs state:
"Returns the maximum re-identification probability on the real dataset from the synthetic dataset.
For each dataset partition, we report the maximum ratio of unique sensitive information between the real dataset and in the synthetic dataset." 
However, the delta_presence method does not require inputting the Sensitive Attributes and generally delta-presence relies on the comparison with a public dataset.

It would be great to have a paper reference, to know how sensitive attributes are chosen and what is the public dataset that this algo refers to.

Thank you for the excellent work.

**Comment by robsdavis** [comment]
Thank you for taking the time to report this issue and for your kind words about our library! I appreciate the opportunity to clarify the functionality of the DeltaPresence privacy metric and address your concerns.

# Clarifying the DeltaPresence Metric
## Sensitive Attributes Handling:

You're correct in observing that the delta_presence method does not require explicitly inputting sensitive attributes as parameters. Instead, it relies on the dataloader's inherent metadata to identify sensitive features. In the provided [code snippet](https://github.com/vanderschaarlab/synthcity/blob/main/src/synthcity/metrics/eval_privacy.py#L241):

```python
features = _utils.get_features(X_gt, X_gt.sensitive_features)
```
The method accesses X_gt.sensitive_features, which is part of the dataloader object. This means that when preparing your dataloader, you should specify which features are considered sensitive. Here's an example of how you might define sensitive features:

```python
from synthcity.plugins.core.dataloader import GenericDataLoader

# Assuming 'age' and 'salary' are sensitive attributes
sensitive_features = ['age', 'salary']
loader = GenericDataLoader(
    X,
    target_column="target",
    sensitive_columns=sensitive_features,
)
```

If sensitive features are not specified, the metric will default to using all features.

## Definition of the metric
The DeltaPresence metric is designed to evaluate the re-identification risk by comparing the distribution of non-sensitive features between the real (ground truth) dataset and the synthetic dataset. It calculates the maximum ratio of unique non-sensitive information occurrences across various dataset partitions, providing an empirical measure of privacy risk.


## Public dataset reference
Here I think any reference to a 'public dataset' will just mean a ground truth dataset.

Thank You
Your feedback is invaluable in helping improve the library and ensuring that it meets the needs of its users. If you have any further questions or suggestions, please don't hesitate to reach out!

Best regards,
Rob Davis
